cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_on_termination_script.sh.erb

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Minor: We can shorten the file name as 'login_nodes_on_termination.sh.erb', omitting the term 'script' because it's implicit in the file extension that this file is a script.

Reply:
Done!

Reply:
Also done.-> shorten the `login_nodes_daemon_script.sh.erb` as `login_nodes_daemon.sh.erb`